src/server/authserver/Main.cpp

```diff
@@ -113,7 +113,25 @@ extern int main(int argc, char** argv)
 
     sLog->outString("%s (authserver)", _FULLVERSION);
     sLog->outString("<Ctrl-C> to stop.\n");
-    sLog->outString("Using configuration file %s.", configFile);
+	sLog->outString(" ");
+	sLog->outString("   █████╗ ███████╗███████╗██████╗  ██████╗ ████████╗██╗  ██╗");           
+	sLog->outString("  ██╔══██╗╚══███╔╝██╔════╝██╔══██╗██╔═══██╗╚══██╔══╝██║  ██║");           
+	sLog->outString("  ███████║  ███╔╝ █████╗  ██████╔╝██║   ██║   ██║   ███████║");           
+	sLog->outString("  ██╔══██║ ███╔╝  ██╔══╝  ██╔══██╗██║   ██║   ██║   ██╔══██║");           
+	sLog->outString("  ██║  ██║███████╗███████╗██║  ██║╚██████╔╝   ██║   ██║  ██║");           
+	sLog->outString("  ╚═╝  ╚═╝╚══════╝╚══════╝╚═╝  ╚═╝ ╚═════╝    ╚═╝   ╚═╝  ╚═╝");           
+	sLog->outString("   ");                                                        
```

Comment:
@ShinDarth I'm not seeing anything wrong there and if you look at the code will see that it does not have these spaces and this standard do not know why GitHub shows these spaces because in my local notebook the code is perfect see for yourself below:
[AuthServer_Main.zip](https://github.com/AzerothCore/azerothcore-wotlk/files/380298/AuthServer_Main.zip)